app/client/components/viewCommon.css

```
@@ -207,9 +210,11 @@
   border-left-color: var(--grist-theme-table-header-border, var(--grist-color-dark-grey));
 }
 
-.column_name.selected {
-  color: var(--grist-theme-table-header-selected-fg, unset);
-  background-color: var(--grist-theme-table-header-selected-bg, var(--grist-color-medium-grey-opaque));
+.column_name.selected > .column_name_selection {
```

```suggestion
.column_name.selected > .selection {
```

It's ok to remove `dom('div.column_name_selection')` from `GridView.js`.